### PR TITLE
Updates and corrections to below pages

### PR DESCRIPTION
**Pull request by MikeJenson**
Corrected link to chord progression generator.
Added link to new Dice Roll Calculator.

### CODE REVIEW

**Review by EnderSmith — DISMISSED** [review]
Thanks for these changes!

File: ChordProgress/ChordProg.htm [hunk 1]
```diff
@@ -56,5 +56,12 @@
       </div>
     </div>
     <div class="test" id="test" style="visibility:hidden"></div>
+    
+    <br>
+    <br>
+    <a href="index.html">
```
Comment (EnderSmith) [inline ChordProgress/ChordProg.htm]: i think this needs to be a relative link to the folder above it or a full url:
`<a href="../index.html">`

File: clrKnit/clrKnit.html [hunk 2]
```diff
@@ -13,6 +13,12 @@
         <div class="btnFrame" id="btnFrame">
         </div>
     </div>
-
+    <br>
+    <br>
+    <a href="index.html">
```
Comment (EnderSmith) [inline clrKnit/clrKnit.html]: same thing, relative or full url:
`<a href="../index.html">`

File: diceRoll/diceRoll.html [hunk 3]
```diff
@@ -104,5 +104,12 @@
         </div>
       </div>
     </div>
+    <br>
+    <br>
+    <a href="index.html">
```
Comment (EnderSmith) [inline diceRoll/diceRoll.html]: same thing, relative or full url:
`<a href="../index.html">`

File: index.html [hunk 4]
```diff
@@ -26,10 +26,16 @@
       </div>
     </a>
     <br>
-    <a href="ChordProgress/ChordProgress.html">
+    <a href="ChordProgress/ChordProg.html">
```
Comment (EnderSmith) [inline index.html:29]: thanks for finding that!

File: payclock.html [hunk 5]
```diff
@@ -39,5 +39,12 @@
       <span id="restart_btn" onclick="restart()" class=""></div>
       <span id="clipboard_btn" onclick="clipboard()" class=""></div>
     </div>
+    <br>
+    <br>
+    <a href="index.html">
```
Comment (EnderSmith) [inline payclock.html]: (this one is correct. I need to move into its own folder like the others!)

**Review by EnderSmith — CHANGES_REQUESTED** [review]
Thanks for contributing!
I noticed that the back links need to be relative to the parent folder.

**Comment by MikeJenson** [comment]
I believe you're correct on that actually, I've not worked much with html in folders, I think that having it link back to the full URL would be best just to make it "future proof" so you wouldn't need to change if you change the folder layout

**Comment by MikeJenson** [comment]
I've run through and made all requested changes. I have also moved the "payclock" files into their own folder.

**Review by MikeJenson — COMMENTED** [review]
These requests have now been fulfilled.

**Review by EnderSmith — APPROVED** [review]
It's possible we'll have to go back and add direct links to the back home buttons to keep them futureproof

**Pull request merged**